Tests/LibWeb/test-web/Fuzzy.cpp

```diff
@@ -27,8 +27,10 @@ bool fuzzy_screenshot_match(URL::URL const& reference, Gfx::Bitmap const& bitmap
             return fuzzy_match.reference.value().equals(reference);
         return true;
     });
-    if (!fuzzy_match.has_value())
+    if (!fuzzy_match.has_value()) {
+        warnln("Screenshot mismatch: pixel error count {}, with maximum error {}. (No fuzzy config defined)", diff.pixel_error_count, diff.maximum_error);
         return diff.identical;
```

Comment:
Ooh we can just do `return false;` here.

Reply:
Good point! Done